.github/workflows/sphinx.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       - name: Build Documentation
         run: |
           pip install -e .[docs]
+          pip install -r docs/requirements.txt  # redundant?
```

Comment:
Yep redundant

Reply:
The full question would be: does the sphinx build automatically find docs/requirements.txt

Unfortunately there isn't a good way to make sure our pip "docs" extra matches docs/requirements.txt